WordPressAuthenticator/Signin/AppleAuthenticator.swift

```diff
@@ -154,6 +165,11 @@ private extension AppleAuthenticator {
         delegate?.showWPComLogin(loginFields: loginFields)
     }
     
+    func show2FA() {
+        WordPressAuthenticator.track(.signupSocialToLogin, properties: ["source": "apple"])
```

Comment:
Hey @frosty . Thanks for noticing this. I misunderstood what `signupSocialToLogin` was for, and it shouldn't even be here at all. It is logged correctly elsewhere. I'll remove it in a follow up PR. Thanks!

Reply:
I looked into this some more, and logging this is actually correct.

`signupSocialToLogin` is used when the user starts by selecting Sign Up, selects an existing account, and automatically gets put in the Log In flow.

Both Apple and Google are already logging this with non-2FA accounts.

The difference here is Google does not handle signing up with an existing 2FA account. That path fails (and from what I can tell, always has). So that path doesn't currently have the `signupSocialToLogin` Track as it doesn't actually do that. 

This SIWA method `show2FA` is called when an existing 2FA account is selected, and the user gets directed to log in instead. So logging `signupSocialToLogin` here is appropriate.